integtest/Makefile

```diff
@@ -298,13 +384,28 @@ define GREP=
 	# grep for a string in a file, outputting the whole file if there isn't
 	# a match.
 	[ -e $(2) ] || { \
-		echo "can't find $(2)"; \
+		echo "can't find \'$(2)\'"; \
 		ls $$(dirname $(2)); \
 		false; \
 	}
 	grep $(1) $(2) > /dev/null || { \
-		echo "Couldn't" find $(1) in $(2):; \
+		echo "Couldn't" find \'$(1)\' in \'$(2)\':; \
 		cat $(2); \
 		false; \
 	}
 endef
+
+define GREP_V=
+	# grep for a string in a file, outputting the whole file if there *is*
+	# a match.
+	[ -e $(2) ] || { \
```

Comment:
`assert_not_in_file`. But I need to print the file so I can figure out what is going on.

Reply:
Do you think `assert_not_in_file` is a better name for this? I'm so used to thinking about `grep -v` but the output on failure is different.

Reply:
I don't think it's a big deal, but after reading the function name and the comment, I still wasn't sure which part of the function was the core and which was the side-effect. It's not a big public API, so just tweaking the comment a bit would probably be fine, too.

I don't _love_ the name `GREP_V`, because it reveals the implementation, but again, this a Makefile, not some hallowed, live-with-it-forever public API.

My question was mostly to satisfy my curiosity, rather than to demand changes.